session_security/middleware.py

```diff
@@ -51,7 +52,12 @@ def get_expire_seconds(self, request):
 
     def process_request(self, request):
         """ Update last activity time or logout. """
-        if not request.user.is_authenticated():
+        if django.VERSION < (1, 10):
```

Comment:
Should we not check < (2, 0) ?

Reply:
No, the [methods have been deprecated since 1.10](https://docs.djangoproject.com/en/1.10/releases/1.10/#user-is-auth-anon-deprecation), they are removed in 2.0.

Reply:
Ok, i'm all for making dj2 a requirement for the next release of dss